app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -3648,7 +3731,7 @@ class BrowserTabFragment :
                 omnibar.renderOmnibarViewState(viewState)
 
                 lastSeenBrowserViewState?.let {
-                    omnibar.renderBrowserViewState(it, tabDisplayedInCustomTabScreen)
+                    // omnibar.renderBrowserViewState(it, tabDisplayedInCustomTabScreen)
```

Comment:
Should this be commented out?

Reply:
it should be deleted, this is not necessary and it’s redundant. for all and new omnibars.